.github/workflows/CI.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
We don't need the full ubuntu image. It's too large to just run node / jest tests.

There's node specific ones. https://hub.docker.com/_/node/